### PR TITLE
feat: tokenPocket support

### PR DESCRIPTION
**Pull request by chendatony31**

### CODE REVIEW

File: packages/rainbowkit/src/wallets/walletConnectors/tokenPocketWallet/tokenPocketWallet.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+import { InjectedConnector } from 'wagmi/connectors/injected';
+import { Chain } from '../../../components/RainbowKitProvider/RainbowKitChainContext';
+import { isMobile } from '../../../utils/isMobile';
+import { Wallet } from '../../Wallet';
+import { getWalletConnectConnector } from '../../getWalletConnectConnector';
+
+export interface TokenPocketWalletOptions {
+  projectId?: string;
```
Comment (DanielSinclair) [inline packages/rainbowkit/src/wallets/walletConnectors/tokenPocketWallet/tokenPocketWallet.ts:9]: `projectId` is now required after 1.0.2

File: packages/rainbowkit/src/wallets/walletConnectors/tokenPocketWallet/tokenPocketWallet.ts [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+  return {
+    id: 'tokenPocket',
+    name: 'TokenPocket',
+    iconUrl: async () => (await import('./tokenPocketWallet.svg')).default,
+    iconBackground: '#2980FE',
+    installed: !shouldUseWalletConnect ? isTokenPocketInjected : undefined,
+    downloadUrls: {
+      browserExtension:
+        'https://chrome.google.com/webstore/detail/tokenpocket/mfgccjchihfkkindfppnaooecgfneiii',
```
Comment (DanielSinclair) [inline packages/rainbowkit/src/wallets/walletConnectors/tokenPocketWallet/tokenPocketWallet.ts:30]: Should use `chrome` instead and provide a fallback site as `browserExtension`

File: packages/rainbowkit/src/wallets/walletConnectors/tokenPocketWallet/tokenPocketWallet.svg
Comment (DanielSinclair) [inline packages/rainbowkit/src/wallets/walletConnectors/tokenPocketWallet/tokenPocketWallet.svg:1]: Please minify, and resize to match the other wallet connectors icons `width="28" height="28" viewBox="0 0 28 28"`. This helps bundle sizes.

**Review by DanielSinclair — CHANGES_REQUESTED** [review]

**Comment by chendatony31** [comment]
Hi, @DanielSinclair  all issues have been fixed.

**Review by DanielSinclair — APPROVED** [review]

**Pull request merged**